pynrfjprog/__init__.py

```diff
@@ -3,4 +3,31 @@
 
 """
 
-__version__ = "10.19.0" 
+__version__ = "10.23.0" 
+
+import sys
+from pathlib import Path
+import stat
+import logging
+
+
+def set_worker_executable(on_load=False):
+    os_name = sys.platform.lower()
+    if os_name.startswith("dar") or os_name.startswith("linux"):
```

Comment:
@python-journeyman Do you know why we are checking if the os_name starts with "dar" and not "darwin"? Is there a special case we need to be aware of?

Reply:
No special cases that I know of, but it wouldn't make a difference anyway. Would just leave it.